index.html

```diff
@@ -183,14 +202,14 @@ <h2 class="title-post"> Profesional Art Printing Data </h2>
             <a class="orange-button" href="#">
                 See Project
             </a>                        
```

Comment:
As requested by the previous reviewer, kindly fix the size of this button to make the UI more attractive.

**See here**

<img width="476" alt="bad button" src="https://user-images.githubusercontent.com/84930355/206859164-90878097-2149-43ea-a6f8-e1de5174d75d.png">